--- process-old.py	2024-11-26 16:20:27.856796223 +0000
+++ process.py	2024-11-26 16:19:40.522385074 +0000
@@ -148,6 +148,9 @@
             if self.args.noise_w:
                 piper_args.extend(["--noise-w", str(self.args.noise_w)])
 
+            if self.args.cuda:
+                piper_args.extend(["--cuda"])
+
             _LOGGER.debug(
                 "Starting piper process: %s args=%s", self.args.piper, piper_args
             )
