--- __main__-old.py	2024-11-26 16:26:59.644200503 +0000
+++ __main__.py	2024-11-26 16:19:53.459497442 +0000
@@ -65,6 +65,11 @@
         action="store_true",
         help="Download latest voices.json during startup",
     )
+    parser.add_argument(
+        "--cuda",
+        action="store_true",
+        help="Use GPU"
+    )
     #
     parser.add_argument("--debug", action="store_true", help="Log DEBUG messages")
     args = parser.parse_args()
